.github/workflows/check-pr-title.yml

```diff
@@ -1,24 +1,36 @@
 name: "Check PR Title"
 on:
   pull_request:
-    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled, converted_to_draft, edited]
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+      - labeled
+      - unlabeled
+      - converted_to_draft
+      - edited
+
+permissions:
+  contents: read
+  pull-requests: read
```

Comment:
Same thing here, we don't want workflows to have additional permissions unless they are needed.